src/Commands/Self/Plugin/MigrateCommand.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Self\Plugin;
+
+/**
+ * Migrates Terminus 2 plugins.
+ *
+ * @package Pantheon\Terminus\Commands\Self\Plugin
+ */
+class MigrateCommand extends InstallCommand
```

Comment:
Commands shouldn't override other command classes. Refactor the methods you need from InstallCommand into a base class or utility class.

Reply:
Didn't notice this before.